electron/updater.ts

```diff
@@ -1,5 +1,6 @@
 import { app, dialog } from 'electron'
 import { autoUpdater } from 'electron-updater'
+import { logInfo } from 'logger'
```

Comment:
This auto-import broke heroic because it's trying to find a npm module with logger on it. Change it to the right path.
Error: 
```JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'logger'
Require stack:
- /tmp/.mount_HeroicglEprB/resources/app.asar/build/updater.js
- /tmp/.mount_HeroicglEprB/resources/app.asar/build/protocol.js
- /tmp/.mount_HeroicglEprB/resources/app.asar/build/main.js
- 
    at Module._resolveFilename (internal/modules/cjs/loader.js:887:15)
    at Function.n._resolveFilename (electron/js2c/browser_init.js:257:1128)
    at Module._load (internal/modules/cjs/loader.js:732:27)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12913)
    at Module.require (internal/modules/cjs/loader.js:959:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/tmp/.mount_HeroicglEprB/resources/app.asar/build/updater.js:6:18)
    at Module._compile (internal/modules/cjs/loader.js:1078:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1108:10)
    at Module.load (internal/modules/cjs/loader.js:935:32)```

Reply:
Fixed